### PR TITLE
Update types for "react-input-mask" v2.0.3

### PR DESCRIPTION
**Pull request by dima7a14**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/sanniassin/react-input-mask>
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/react-input-mask/v2/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
+// Type definitions for react-input-mask 2.0.3
+// Project: https://github.com/sanniassin/react-input-mask
+// Definitions by: Alexandre Paré <https://github.com/apare>
+//                 Dima Danylyuk <https://github.com/dima7a14>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.9.2
+
+declare module 'react-input-mask' {
```
Comment (plantain-00) [inline types/react-input-mask/v2/index.d.ts]: The module declaration seems can be removed

**Review by plantain-00 — CHANGES_REQUESTED** [review]
Can you move existing files to `v1/` and replaced with your files: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version

File: types/react-input-mask/v2/index.d.ts (hunk 1 above)
Comment (dima7a14) [reply types/react-input-mask/v2/index.d.ts]: Thank you. But without module declaration I have ts error: *Try `npm install @types/react-input-mask` if it exists or add a new declaration (.d.ts) file containing `declare module 'react-input-mask';`*.

Comment (plantain-00) [reply types/react-input-mask/v2/index.d.ts]: The module declaration is used when the definition is not in `react-input-mask` package and `@types/react-input-mask` package, your code will be in `@types/react-input-mask`, so not needed.
You can check other packages' types

Comment (dima7a14) [reply types/react-input-mask/v2/index.d.ts]: You are right, thanks.

**Review by plantain-00 — APPROVED** [review]

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Comment by dima7a14** [comment]
Yes, you're right, I'll add this interface. Thank you.

**Pull request merged**